Lab2/html/formas.circulo.html

```diff
@@ -15,7 +15,7 @@
     <script src="../js/bootstrap.bundle.min.js"></script> 
     <script src="../js/p5.min.js"></script> 
     <script src="../js/formas.circulo.js"></script> 
-    <script src="../js/algoritmos.js"></script>
+    <script src="../js/algoritmossss.js"></script>
```

Comment:
I think, it's better make this changes: 

```suggestion
    <script src="../js/algoritmos.js"></script>
```